Sources/VLCMovieViewController.m

```diff
@@ -44,7 +45,7 @@
 #define DEFAULT_FOV 80.f
 #define MAX_FOV 150.f
 #define MIN_FOV 20.f
-#define NEW_UI 0
+#define NEW_UI 1
```

Comment:
I don't think this change should be committed.
Additionally, it seems that it doesn't compile with the "old" UI.

Reply:
this should be 0 I agree